.circleci/config.yml

```diff
@@ -75,7 +75,7 @@ jobs:
     steps:
       - attach_workspace:
           at: ./
-      - run: npx ng build --configuration ci --base-href //d39se6ub293skr.cloudfront.net/branch/<< pipeline.git.branch >> --progress=false
+      - run: npx ng build --configuration ci --base-href /branch/<< pipeline.git.branch >> --progress=false
```

Comment:
:warning: Given the new configuration of the common build step, I think it will break deployment of master

I think we should have different build steps of branches and for master
Another solution would be to give a base-href value conditioned by the branch name